lib/sprockets/sassc_importer.rb

```diff
@@ -0,0 +1,186 @@
+require 'sprockets/autoload'
+
+module Sprockets
+  class SasscImporter < Autoload::SassC::Importer
+    class Extension
+      attr_reader :postfix
+
+      def initialize(postfix=nil)
+        @postfix = postfix
+      end
+
+      def import_for(full_path, parent_dir, options)
+        SassC::Importer::Import.new(full_path)
+      end
+    end
+
+    class CSSExtension
+      def postfix
+        ".css"
+      end
+
+      def import_for(full_path, parent_dir, options)
+        import_path = full_path.gsub(/\.css$/,"")
+        SassC::Importer::Import.new(import_path)
+      end
+    end
+
+    class CssScssExtension < Extension
+      def postfix
+        ".css.scss"
+      end
+
+      def import_for(full_path, parent_dir, options)
+        source = File.open(full_path, 'rb') { |f| f.read }
```

Comment:
Let’s use `File.read` here and in other similar spots instead since it’s a better use of the standard library.